webapp/src/main/java/com/box/l10n/mojito/service/machinetranslation/TextUnitTranslationGroupDTO.java

```diff
@@ -8,7 +9,7 @@
  *
  * @author garion
  */
-public class TextUnitTranslationGroupDTO {
+public class TextUnitTranslationGroupDTO implements Serializable {
```

Comment:
is that for the cache?

Reply:
Yes, to serialize and store the response as a binary blob.